manifests/instance.pp

```diff
@@ -167,14 +182,14 @@
         high_port => $port,
```

Comment:
Line 177: if we are using puppet/selinux, how are we assured the necessary selinux packages are installed?

Reply:
@lnemsick-simp I'll remove it.